packages/big-design/src/components/MultiSelect/types.ts

```diff
@@ -7,6 +7,7 @@ import { SelectOptionGroup } from '../Select/types';
 
 interface BaseSelect extends Omit<React.HTMLAttributes<HTMLInputElement>, 'children'> {
   action?: SelectAction;
+  autoComplete?: string;
```

Comment:
Can you check that this is not on `React.HTMLAttributes<HTMLInputElement>`?

Reply:
It isn't

Reply:
Ah makes sense. We'd probably want to do this for the `Form` component too then.

Reply:
I assume it defaults to `on` on the form?

Reply:
Yep

Reply:
`FormHTMLAttributes` does have `autoComplete`

Reply:
Hmm, I could swap the type of the Selects for `<React.InputHTMLAttributes<HTMLInputElement>` which do include `autoComplete` and a bunch of others

Reply:
🤔 

Only thing we need to make sure if we do, is that the types we say we allow (for example, valid input attributes like any `aria-` ones, or the `multiple` attribute) are actually passed in to the underlying component. We don't want our types to say we have something that our component doesn't actually implement.

[Full list of input attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes)

Reply:
Lets keep as is since I don't want to expose all those to the user.